packages/launcher-component/src/hubs/src-repository-hub.tsx

```diff
@@ -36,7 +40,26 @@ export const SrcRepositoryHub: FormHub<SrcRepositoryFormValue> = {
       </OverviewComplete>
     );
   },
-  Form: props => (
+  Form: props => {
+    const convert = object => {
```

Comment:
Correct me if I am wrong, but those could converter could be part of the `BuildImageAnalyzerLoader`?

Reply:
sure, just wanted it to get it working first